src/languages/turtle.js

```diff
@@ -48,7 +48,7 @@ function(hljs) {
   };
 
   var LANGTAG = {
-    begin: /@[a-zA-Z]+([a-zA-Z0-9-]+)*/,
+    begin: /@[a-zA-Z]+(-[a-zA-Z0-9]+)*/,
```

Comment:
Did this fix a backtrack?

Reply:
Not fixing a backtrack though this is actually the cause of the exponential backtracking. Both versions cause it.

The [spec for turtle](https://www.w3.org/TR/turtle/) actually has it as:

```
[144s] LANGTAG '@' [a-zA-Z]+ ('-' [a-zA-Z0-9]+)*
```

Which I think makes sense - we should match `@en-us` for example but not `@en--us`.

Reply:
> Both versions cause it.

Are you sure? Can you past the exact error with help text?

Reply:
Looks like the new one did fix a backtrack actually! My mistake.